Kokoro CI not running because system-test dir has no tests to run

When kokori ci runs, the following error is generated:
```
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/code-suggester@0.1.0 prepare /tmpfs/src/github/code-suggester
> npm run compile


> @google-cloud/code-suggester@0.1.0 compile /tmpfs/src/github/code-suggester
> tsc -p .

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN notsup Unsupported engine for watchpack-chokidar2@2.0.0: wanted: {"node":"<8.10.0"} (current: {"node":"10.21.0","npm":"6.14.4"})
npm WARN notsup Not compatible with your version of node/npm: watchpack-chokidar2@2.0.0
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/watchpack-chokidar2/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1073 packages from 871 contributors and audited 1076 packages in 29.873s

52 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details

> @google-cloud/code-suggester@0.1.0 system-test /tmpfs/src/github/code-suggester
> c8 mocha build/system-test

Error: No test files found: "build/system-test"
-----------------------------------|---------|----------|---------|---------|-------------------
File                               | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------------------------------|---------|----------|---------|---------|-------------------
All files                          |    3.08 |     7.14 |       0 |    3.08 |
 code-suggester                    |   48.89 |       20 |       0 |   48.89 |
  .mocharc.js                      |   78.57 |       25 |     100 |   78.57 | 20-21,23-24,26-27
  .prettierrc.js                   |       0 |        0 |       0 |       0 | 1-17
 code-suggester/src                |       0 |        0 |       0 |       0 |
  default-options-handler.ts       |       0 |        0 |       0 |       0 | 1-41
  index.ts                         |       0 |        0 |       0 |       0 | 1-84
 code-suggester/src/github-handler |       0 |        0 |       0 |       0 |
  branch-handler.ts                |       0 |        0 |       0 |       0 | 1-89
  commit-and-push-handler.ts       |       0 |        0 |       0 |       0 | 1-188
  fork-handler.ts                  |       0 |        0 |       0 |       0 | 1-51
  index.ts                         |       0 |        0 |       0 |       0 | 1-4
  pr-handler.ts                    |       0 |        0 |       0 |       0 | 1-56
 code-suggester/src/logger         |       0 |        0 |       0 |       0 |
  index.ts                         |       0 |        0 |       0 |       0 | 1-23
 code-suggester/src/types          |       0 |        0 |       0 |       0 |
  index.ts                         |       0 |        0 |       0 |       0 | 1-134
-----------------------------------|---------|----------|---------|---------|-------------------
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/code-suggester@0.1.0 system-test: `c8 mocha build/system-test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @google-cloud/code-suggester@0.1.0 system-test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2020-07-24T16_40_20_018Z-debug.log
cleanup


[ID: 9122827] Build finished after 72 secs, exit value: 1


Warning: Permanently added 'localhost' (ECDSA) to the list of known hosts.
[09:40:21] Collecting build artifacts from build VM
Build script failed with exit code: 1
```
Create a system-test dir with a test file so kokoro passes.

--- a/system-test/main.test.ts
+++ b/system-test/main.test.ts
@@ -1,0 +1,22 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {assert} from 'chai';
+import {describe, it} from 'mocha';
+
+describe('System test', async () => {
+  it('has a test case', () => {
+      assert.isOk(true);
+  });
+});

--- a/system-test/main.test.ts
+++ b/system-test/main.test.ts
@@ -15,8 +15,8 @@
 import {assert} from 'chai';
 import {describe, it} from 'mocha';
 
-describe('System test', async () => {
+describe('System test', () => {
   it('has a test case', () => {
-      assert.isOk(true);
+    assert.isOk(true);
   });
 });

